src/ui-shell/sidenav.component.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, HostBinding, ViewEncapsulation } from "@angular/core";
```

Comment:
To fix the currently present error in preview.

```suggestion
import {
	Component,
	HostBinding,
	Input,
	ViewEncapsulation
} from "@angular/core";
```